README.md

```diff
@@ -687,6 +699,15 @@ RULE:   $ rmadison -c main {all,packages,that,look,like,duplicates}
 RULE: If any of them are reported to be in main check in detail if they cover
 RULE: indeed the same use case as the package this MIR is about.
 TODO: There is no other package in main providing the same functionality.
+RULE: No matter how useful a rationale is and how unique a package might be
+RULE: it will need an owning team that is willing and able to spend the time
+RULE: to maintain it well for the benefit of all Ubuntu users and use cases.
+TODO: A team is committed to own long term maintenance of this package.
+RULE: We ask the reporters to state a reasoning, but a MIR team member should
+RULE: try to judge if this is any good for Ubuntu and its users.
```

Comment:
the word "any" is inaccurate and unnecessary

Reply:
ack